server/datastore/mysql/migrations/tables/20220503134048_AddCVEsTable.go

```diff
@@ -0,0 +1,31 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220503134048, Down_20220503134048)
+}
+
+func Up_20220503134048(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE cves (
+    cve varchar(20) PRIMARY KEY,
+    cvss_score double,
```

Comment:
Do we really need double precision?

Reply:
What concern do you have for using `double`? Would you suggest using `float` instead (single precision)?